File: src/main/java/com/cocktailsguru/app/cocktail/domain/ingredient/AlcoIngredient.java
package com.cocktailsguru.app.cocktail.domain.ingredient;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;

@EqualsAndHashCode(callSuper = true)
@Entity(name = "coctail_ingred_alco")
@Data
@NoArgsConstructor
public class AlcoIngredient extends BaseIngredient {

    private Integer voltage;
}

File: src/main/java/com/cocktailsguru/app/ingredient/domain/AlcoIngredient.kt
package com.cocktailsguru.app.ingredient.domain

import javax.persistence.Entity

@Entity(name = "coctail_ingred_alco")
data class AlcoIngredient(
        override val id: Long,
        override var name: String,
        override var nameGrouped: String,
        override var description: String,
        override var imageName: String,
        override var numShowed: Int,
        override var ingredientType: IngredientType,
        var voltage: Int) : BaseIngredient(id, name, nameGrouped, description, imageName, numShowed, ingredientType)